samples/matter/light_bulb/CMakeLists.txt

```diff
@@ -12,6 +12,11 @@ get_filename_component(MATTER_MODULE_ROOT $ENV{ZEPHYR_BASE}/../modules/lib/matte
 set(mcuboot_KCONFIG_ROOT ${MATTER_MODULE_ROOT}/config/nrfconnect/chip-module/Kconfig.mcuboot.root)
 set(multiprotocol_rpmsg_KCONFIG_ROOT ${MATTER_MODULE_ROOT}/config/nrfconnect/chip-module/Kconfig.multiprotocol_rpmsg.root)
 
+# Propagate nRF7002 EK shield configuration to the child image to include Wi-Fi coexistence definitions.
+if (SHIELD STREQUAL "nrf7002_ek")
```

Comment:
Right, but we don't really support any other shields AFAIK in these samples. Anyway, if there's really no non-controversial way to set this automatically for a user instead of forcing them to learn about all inner dependencies, I'll give up and remove this completely. Especially given that @markaj-nordic says the coex doesn't work anyway.

Reply:
This code has been removed. Resolving.